lib/k8s-e2e-tests/tests/vector.rs

```diff
@@ -196,15 +85,28 @@ async fn logs() -> Result<(), Box<dyn std::error::Error>> {
     framework
         .wait_for_rollout(
             &namespace,
-            &format!("daemonset/{}", agent_override_name),
+            &format!("statefulset/aggregator-vector"),
```

Comment:
Theoretically this could be not hardcoded, but I didn't feel it was worth investing additional time now with the possible larger scale changes in Q1